package-lock.json

```diff
@@ -1,11626 +1,8 @@
 {
   "name": "webpack-starter-kit",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
This has happened to me in the past, I dont think that json file should have changed to "lockfileVersion":1 

Does anyone else have any insight on this?